Add interfaces/trusted-types.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/797d4c5/ed/idl/trusted-types.idl
Build: https://travis-ci.org/w3c/webref/builds/219343667

stephenmcgruer reviewed (changes requested):
https://github.com/web-platform-tests/wpt/pull/25772 should handle this (idl file previously added manually, it was just marked tentative)

autofoolip committed:
Add interfaces/trusted-types.idl

Source: https://github.com/w3c/webref/blob/797d4c5/ed/idl/trusted-types.idl
Build: https://travis-ci.org/w3c/webref/builds/219343667
